.github/workflows/main-build.yml

```diff
@@ -0,0 +1,39 @@
+name: main-build
+
+on:
+  push:
+    branches:
+      - main
+    paths-ignore:
+      - "docs/"
+      - "**.md"
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+      # Cloning
+      - uses: actions/checkout@v3
+
+      # Setup and Caching
+      - name: Use latest Node.js LTS version
+        uses: actions/setup-node@v3
+        with:
+          node-version: lts/*
+
+          # NPM Cache using all package-lock files as hash
+          cache: 'npm'
+          cache-dependency-path: '**/package-lock.json'
```

Comment:
We now only have one package-lock file right?
Can this be changed now or do we still need to target all possible package lock files?

Reply:
Just in case we can keep this, since it doesn't hurt.